src/PanelTraits/Search.php

```diff
@@ -133,38 +133,53 @@ public function getRowViews($entry)
      */
     public function getCellView($column, $entry)
     {
-        // if column type not set, show as text
-        if (! isset($column['type'])) {
-            return \View::make('crud::columns.text')
-                            ->with('crud', $this)
-                            ->with('column', $column)
-                            ->with('entry', $entry)
-                            ->render();
-        } else {
-            // if the column has been overwritten show that one
-            if (view()->exists('vendor.backpack.crud.columns.'.$column['type'])) {
-                return \View::make('vendor.backpack.crud.columns.'.$column['type'])
-                                ->with('crud', $this)
-                                ->with('column', $column)
-                                ->with('entry', $entry)
-                                ->render();
-            } else {
-                // show the column from the package
-                if (view()->exists('crud::columns.'.$column['type'])) {
-                    return \View::make('crud::columns.'.$column['type'])
-                                    ->with('crud', $this)
-                                    ->with('column', $column)
-                                    ->with('entry', $entry)
-                                    ->render();
-                } else {
-                    return \View::make('crud::columns.text')
-                                    ->with('crud', $this)
-                                    ->with('column', $column)
-                                    ->with('entry', $entry)
-                                    ->render();
-                }
+        return $this->renderCellView($this->getCellViewName($column), $column, $entry);
+    }
+
+    /**
+     * Get the name of the view to load for the cell
+     * @param $column
+     * @return string
+     */
+    private function getCellViewName($column)
+    {
+        // return custom column from view attribute
+        if (isset($column['view']) && isset($column['type'])) {
+            return $column['view'] . '.' . $column['type'];
```

Comment:
This naming is misleading in my opinion. If I set a view option, I would expect giving the whole view identifier instead only the prefix.